### PR TITLE
feat: 🎸 fetch all CustomClaimType(s)

### PR DESCRIPTION
**Pull request by sansan**
### Description

adds a method to fetch all registered CustomClaimType(s)

### JIRA Link

https://polymesh.atlassian.net/browse/DA-937

### Checklist

- [x] Updated the Readme.md (if required) ?

### CODE REVIEW

File: src/api/client/Claims.ts [hunk 1]
```diff
@@ -531,4 +539,50 @@ export class Claims {
 
     return { id, name: bytesToString(customClaimTypeIdOpt.value) };
   }
+
+  /**
+   * Retrieve all claims issued about an Identity, grouped by claim issuer
```
Comment (prashantasdeveloper) [inline src/api/client/Claims.ts:544]: Description needs to be updated to specify custom claims

**Review by prashantasdeveloper — COMMENTED** [review]
LGTM. Left a small nit comment around docs

**Comment by prashantasdeveloper** [comment]
@sansan 
You would also have to raise the minimum SQ version to version where custom claims were added

**Review by polymath-eric — APPROVED** [review]

**Pull request merged**